ui/components/app/multilayer-fee-message/multi-layer-fee-message.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState, useEffect } from 'react';
```

Comment:
This is the file responsible for displaying the totals when on a "multilayer" network.

![Screenshot from 2021-11-11 02-15-26](https://user-images.githubusercontent.com/7499938/141244406-6e88cfa3-71b5-4847-8a24-0b8be9393205.png)